internal/integration/sample_api/oas_handlers_gen.go

```diff
@@ -2164,8 +2164,7 @@ func (s *Server) handleSecurityTestRequest(args [0]string, w http.ResponseWriter
 			ID:   "securityTest",
 		}
 	)
-	sctx, err := s.securityAPIKey(ctx, "SecurityTest", r)
-	if err != nil {
```

Comment:
Usually, we change all generated files in a one separate commit. Could you move these changes to a separate commit, please?

Reply:
Done @tdakkota